bertopic/_bertopic.py

```diff
@@ -4132,21 +4131,43 @@ def _reduce_to_n_topics(self, documents: pd.DataFrame) -> pd.DataFrame:
 
         # Cluster the topic embeddings using AgglomerativeClustering
         if version.parse(sklearn_version) >= version.parse("1.4.0"):
-            cluster = AgglomerativeClustering(self.nr_topics - self._outliers, metric="precomputed", linkage="average")
+            cluster = AgglomerativeClustering(self.nr_topics - self._outliers, metric="precomputed", linkage="complete")
         else:
-            cluster = AgglomerativeClustering(self.nr_topics - self._outliers, affinity="precomputed", linkage="average")
+            cluster = AgglomerativeClustering(self.nr_topics - self._outliers, affinity="precomputed", linkage="complete")
```

Comment:
both clauses are same?

Reply:
The parameter is `metric` vs `affinity`.